assets/prometheus.json

```diff
@@ -3,17 +3,19 @@
   "description": "Lua obfuscation using Prometheus.",
   "id": "prometheus",
   "images": {
-    "hero": "",
-    "thumb": ""
+    "hero": "prometheus-hero.png",
+    "thumb": "prometheus-thumb.png"
   },
   "library_url": "https://github.com/defold/extension-prometheus/archive/refs/heads/master.zip",
   "license": "MIT License",
   "name": "Prometheus",
-  "platforms": [],
+  "platforms": [
+    "All"
```

Comment:
We should probably go over a bunch of other assets as well and change from a list of platforms to All. Especially the purely Lua based assets.